src/test/java/org/opentripplanner/service/vehiclerental/internal/DefaultVehicleRentalServiceTest.java

```diff
@@ -0,0 +1,38 @@
+package org.opentripplanner.service.vehiclerental.internal;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.service.vehiclerental.model.VehicleRentalStation;
+import org.opentripplanner.service.vehiclerental.model.VehicleRentalVehicle;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+class DefaultVehicleRentalServiceTest {
+
+  @Test
+  void getVehicleRentalStationForEnvelopeShouldExcludeVehicleRentalVehicle() {
+    DefaultVehicleRentalService defaultVehicleRentalService = new DefaultVehicleRentalService();
+
+    VehicleRentalStation vehicleRentalStation = new VehicleRentalStation();
```

Comment:
You can use the `TestVehicleRentalStationBuilder` for this. 

In #5361 I'm also adding a builder for free floating vehicles.

Reply:
done